lib/fluent/plugin/out_aws-elasticsearch-service.rb

```diff
@@ -86,17 +87,22 @@ def credentials(opts)
               }).credentials
             end
           else
+            sts_credentials_region = opts[:sts_credentials_region]
+            if sts_credentials_region.empty?
+              sts_credentials_region = opts[:region]
+            end
```

Comment:
If the default is set to nil instead of empty string , can  this can be changed to ?

```
sts_credentials_region = opts[:sts_credentials_region] || opts[:region]
```

Reply:
Updated the PR take use this syntax.